dist/openshift/cincinnati.yaml

```
@@ -289,3 +290,6 @@ parameters:
   - name: PE_BINARY
     value: /usr/bin/policy-engine
     displayName: Path to policy-engine binary
+  - name: GB_UPSTREAM_FETCH_CONCURRENCY
+    value: 16
```

```suggestion
    value: "16"
```

This causes
```
```
 error: unable to load template file "dist/openshift/cincinnati.yaml": unable to decode "dist/openshift/cincinnati.yaml": v1.Template.Parameters: []v1.Parameter: v1.Parameter.Value: ReadString: expects " or n, but found 1, error found in #10 byte of ...|,"value":16}]}|..., bigger context ...|","name":"GB_UPSTREAM_FETCH_CONCURRENCY","value":16}]}|...
error: unable to locate any template files with name "dist/openshift/cincinnati.yaml"
```
in e2e tests

---

Oh, thanks for catching this.